manifests/0000_26_cloud-controller-manager-operator_03_rbac_provider.yaml

```diff
@@ -187,3 +187,47 @@ subjects:
 - kind: ServiceAccount
   name: cloud-controller-manager
   namespace: openshift-cloud-controller-manager
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
```

Comment:
Those are annotations and not labels @lobziik This is why this resource is not created by CVO. Same thing for other resources in this file.